paraview/src/CloudberryDemo.js

```
@@ -0,0 +1,437 @@
+var cloudberryHost = "ws://localhost:9000/ws"
+var dataModel = { fields: { byTimeResult: { }, byGeoResult: { } }, histogram1D_storage: { 32: { byTimeResult: { }, byGeoResult: { } } }, dirty: true };
+var xhttpState = "initializing";
+var WSServer;
+var xhttp = new XMLHttpRequest();
+
+// timestamp for performance measurement
+var t0;
+var t1;
+
+// select a start date for the query
+var startDate = new Date(2015, 10, 22, 0, 0, 0, 0);
+
+var defaultNonSamplingDayRange = 1500;
+var defaultSamplingDayRange = 1500;
+var defaultSamplingSize = 10;
+
+const bodyElt = document.querySelector('body');
+// '100vh' is 100% of the current screen height
+const defaultHeight = '100vh';
+
+const histogramSelectorContainer = document.createElement('div');
+histogramSelectorContainer.style.position = 'relative';
+histogramSelectorContainer.style.width = '52%';
+histogramSelectorContainer.style.height = defaultHeight;
+histogramSelectorContainer.style.float = 'left';
+bodyElt.appendChild(histogramSelectorContainer);
+
+const fieldSelectorContainer = document.createElement('div');
+fieldSelectorContainer.style.position = 'relative';
+fieldSelectorContainer.style.width = '48%';
+fieldSelectorContainer.style.height = defaultHeight;
+fieldSelectorContainer.style.float = 'left';
+fieldSelectorContainer.style['font-size'] = '10pt';
+bodyElt.appendChild(fieldSelectorContainer);
+
+var countRequest = JSON.stringify({
+    dataset: 'twitter.ds_tweet',
+    global: {
+        globalAggregate: {
+            field: '*',
+            apply: {
+                name: 'count',
+            },
+            as: 'count',
+        },
+    },
+    estimable: true,
+    transform: {
+        wrap: {
+            key: 'totalCount',
+        },
+    },
+});
+
+function getLevel(level) {
+    switch (level) {
+        case 'state': return 'stateID';
+        case 'county': return 'countyID';
+        case 'city': return 'cityID';
+        default:
+            break;
+    }
+    return 'good';
```

I think you should throw an Error here.